HOWTO.md

```diff
@@ -734,3 +736,31 @@ Options:
 
 Please note that in this mode there are some environment variables that are
 still required to be set by the user (`DI_SIGN_KEY_PATH`, `DI_HMAC_KEY_PATH`).
+
+## How to use Features
+  ### How to use `device_specific_store_driver` feature
+  - Using this feaure user can choose to apply different serviceinfo settings to different devices.
+  For that user needs to provide path to device specific file under `device_specific_store_driver` field
+  present in the `serviceinfo_api_server.yml` file.
+  - File name should be name `guid` of the device.
+  - Initialize the device as mentioned in [How to generate an Ownership Voucher and Credential for a Device](#how-to-generate-an-ownership-voucher-ov-and-credential-for-a-device-device-initialization).
+  - Dump the `device-credentials`
+    ```bash
+    fdo-owner-tool dump-device-credential /path/to/device-credentials
+    ```
+  - Note the GUID of the device and create a .yml with same name under directory path `device_specific_store_driver`.
+  - Content of the file should look like:
+    ```bash
+      initial_user:
+        username: user_device_specific
+        sshkeys:
+        - "testkeyperdevice"
+      files: null
+      commands: null
+      diskencryption_clevis: null
+      additional_serviceinfo: null
+      after_onboarding_reboot: false
+    ```
+  - Follow onboarding procedure and this particular device will get serviceinfo settings as mentioned in above file.
```

Comment:
"Follow the onboarding" .. "will get the serviceinfo" "in the file above"/"in the above file"